style.css

```diff
@@ -3,18 +3,18 @@
     margin: 0;
     font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
 }
-#form-content{
+form{
     transform: translate(-50% -50%);
     width: 30%;
     box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
     padding: 2rem;
     border-radius: 15px;
-    color: rgb(32, 17, 17);
+    color: rgb(143, 140, 140);
     position: absolute;
     top: 50%;
     left: 50%;
-    gap: 2px;
-    
+    display: flex;
+    flex-direction: column;
 }
```

Comment:
![image](https://github.com/broad65/form/assets/61404403/6895d0b4-4a05-4e65-ba9c-0dc8472cac06)

The form is not at the center of the page try this:
Add this code at the **Top** and remove it in any part of `form` the CSS selector.
```css
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50% -50%);
```